docs/config-description.md

```diff
@@ -1,452 +1,600 @@
+- **Default value**: `False`
 - **Constraints**: None
- -   **Description**: Enable or disable Prometheus metrics.
-### Key: prometheus.compress_reply
+- **Description**: Enables or disables Prometheus metrics.
+
+### prometheus.compress_reply
+
 - **Required**: True
 - **Type**: boolean
-- **Default value**: False
+- **Default value**: `False`
 - **Constraints**: None
- -   **Description**: Enable or disable compression of Prometheus responses.
-### Key: io_threads
+- **Description**: Enables or disables compression of Prometheus responses.
+
+### io_threads
+
 - **Required**: True
 - **Type**: int
-- **Default value**: 2
-- **Constraints**: The minimum value is `1`. The maximum value is `65535`
- -   **Description**: Number of I/O threads. Value cannot be less than 1
-### Key: subscription_workers
+- **Default value**: `2`
+- **Constraints**: The minimum value is `1`. The maximum value is `65535`.
+- **Description**: The number of input/output (I/O) threads. The value cannot be less than `1`.
+
+### subscription_workers
+
 - **Required**: True
 - **Type**: int
-- **Default value**: 1
-- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`
- -   **Description**: The number of worker threads or processes that are responsible for managing and processing subscription-based tasks from rippled
-### Key: graceful_period
+- **Default value**: `1`
+- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`.
+- **Description**: The number of worker threads or processes that are responsible for managing and processing subscription-based tasks from `rippled`.
+
+### graceful_period
+
 - **Required**: True
 - **Type**: double
-- **Default value**: 10
-- **Constraints**: The value must be a positive double number
- -   **Description**: Number of milliseconds server will wait to shutdown gracefully.
-### Key: cache.num_diffs
+- **Default value**: `10`
+- **Constraints**: The value must be a positive double number.
+- **Description**: The number of milliseconds the server waits to shutdown gracefully.
+
+### cache.num_diffs
+
 - **Required**: True
 - **Type**: int
-- **Default value**: 32
-- **Constraints**: The minimum value is `0`. The maximum value is `65535`
- -   **Description**: Number of diffs to cache. For more info, consult readme.md in etc
-### Key: cache.num_markers
+- **Default value**: `32`
+- **Constraints**: The minimum value is `0`. The maximum value is `65535`.
+- **Description**: The number of diffs to cache. For more information, see the [README.md](../src/etl/README.md) under etl.
+
+### cache.num_markers
+
 - **Required**: True
 - **Type**: int
-- **Default value**: 48
-- **Constraints**: The minimum value is `0`. The maximum value is `65535`
- -   **Description**: Number of markers to cache.
-### Key: cache.num_cursors_from_diff
+- **Default value**: `48`
+- **Constraints**: The minimum value is `0`. The maximum value is `65535`.
+- **Description**: The number of markers to cache.
```

Comment:
Not sure this is "to cache" either. @cindyyan317 do you have better explanation what it does?